README.md

```diff
@@ -14,6 +14,25 @@
 npm install @allegiant/sessions --save
 ```
 
+## Usage
+
+```js
+const Session = require('@allegiant/sessions');
+const App = require('@allegiant/core');
+
+let server = App.create('http://localhost:7000');
+Session.configure(server); // will by default configure needed sesion defaults
```

Comment:
session is mispelled as 'sesion'

Reply:
Also, there isn't a need to do Session.configure directly if you want to store it in the app configuration.. the app will load and bind the module at the right time in this setup:
```js
var server = App.create("https://localhost:7000", { 
    '@allegiant/sessions': {
        enabled: true,
        name: 'id',
        secure: false,
        path: path.resolve(path.join(process.cwd(), 'sessions')),
    }
});
```